app/services/coms.js

```diff
@@ -173,7 +173,7 @@ export default class ComsService extends Service {
   leaveChannel (channel) {
     switch (channel.protocol) {
       case 'XMPP':
-        // TODO implement
+        this.xmpp.leave(channel);
```

Comment:
We can actually remove the protocol switch now, and do something like this:

```js
    this.getServiceForSockethubPlatform(channel.protocol)
        .leave(channel);
```

Reply:
Good point 👍